### PR TITLE
Milestone 4: Testing styling

### PR DESCRIPTION
**Pull request by RichardChileya**
### In this section, I: 

- [ ] Added styling to the App
- [ ] Implemented test
- [ ] Updated the Readme file
- [ ] Fixed linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,46 +27,54 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
```
Comment (mwanawabangona) [inline README.md]: - [x] This image is broken kindly fix it

File: README.md [hunk 2]
```diff
@@ -108,7 +116,7 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link not yet available](https://yourdeployedapplicationlink.com)
```
Comment (mwanawabangona) [inline README.md]: - [x] `required`
- [x] Please add a workable` livedemo` as the requirements states

File: README.md [hunk 3]
```diff
@@ -27,46 +27,54 @@ After you're finished please remove all the comments and instructions!
...
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
+
 - [🙏 Acknowledgements](#acknowledgements)
 - [❓ FAQ](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
```
Comment (mwanawabangona) [inline README.md:65]: - [x] `required`
- [x] I can't not see the recorded video presentation please add it as the requirements state.

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @RichardChileya ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (RichardChileya) [reply README.md:65]: Video has been added

File: README.md (hunk 2 above)
Comment (RichardChileya) [reply README.md]: Demo has been Added

File: README.md (hunk 1 above)
Comment (RichardChileya) [reply README.md]: This has been sorted out

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @RichardChileya ,
# STATUS: APPROVED! 🥇 💯 

done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**